utils/gyb_syntax_support/DeclNodes.py

```diff
@@ -532,6 +532,15 @@
                       'mutableAddressWithNativeOwner',
                       '_read', '_modify'
                    ]),
+             Child('AsyncKeyword', kind='IdentifierToken',
+                   classification='Keyword',
+                   text_choices=['async'], is_optional=True),
+             Child('ThrowsKeyword', kind='Token',
+                   is_optional=True,
+                   token_choices=[
+                       'ThrowsToken',
+                       'RethrowsToken',
+                   ]),
              Child('Parameter', kind='AccessorParameter', is_optional=True),
```

Comment:
Effects specifier should be _after_ `Parameter`. `set(newValue) throws`, not `set throws (newValue)`. It's unfortunate that `test/Parse/effectful_properties.swift` doesn't have cases like it.